docker/secondary-analysis-python/tools/submit/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(name='submit',
+      version='1.0.0.dev1',
+      description='submit',
+      url='http://github.com/HumanCellAtlas/skylab',
+      author='Dave Shiga',
+      author_email='dshiga@broadinstitute.org',
+      license='MIT',
+      packages=['submit', 'tests'],
+      install_requires=[
+          'requests',
+      ],
+      entry_points = {
```

Comment:
I had never seen or heard about this. I like this a lot. How did you come across this? I am interested because you must have searched for something totally different from my normal SO queries!

Reply:
I searched for "pip install callable module" and found some SO posts and blog posts about it.

Reply:
Very cool... thanks!